src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ImageController.php

```diff
@@ -308,11 +307,6 @@ private function getErrorMessages(Exception $e): array
                 'Admin.Notifications.Error'
             ),
             UploadedImageConstraintException::class => [
-                UploadedImageConstraintException::EXCEEDED_SIZE => $this->trans(
```

Comment:
FYI this exception code is still used when thrown in Add/Edit Language handler, so I still left it in the UploadedImageConstraintException class